docs/reference/cheatsheet.md

```diff
@@ -560,6 +560,21 @@ must also include a table title. See example above.
   - ```{important} This is an example
     of an important directive.
     ```
+* - ````md
+    ```{admonition} Title
+    :class: admonition warning
+    text
+    ```
```

Comment:
I think this might be accidental extra stuff?

Reply:
I don't think so. It is part of a big table with 3 columns:

```
  - Syntax
  - Example
  - Result
```

This one is for syntax.